core/door/door_props.py

```diff
@@ -8,17 +8,17 @@
 class DoorProperty(bpy.types.PropertyGroup):
     frame_thickness: FloatProperty(
         name="Frame Thickness",
-        min=0.0,
+        min=0.01,
         max=1.0,
         default=0.1,
         description="Thickness of door Frame",
     )
 
     frame_depth: FloatProperty(
         name="Frame Depth",
-        min=0.0,
+        min=-1.0,
         max=1.0,
-        default=0.0,
+        default=0.05,
```

Comment:
Setting `0.05` as default feels un-natural. The doors and windows now look extremely busy. I think `0.0` is better here, and its nicely midway between min and max

Reply:
This default only looks appropriate on multigroup but not doors and windows.

Reply:
reset default frame depth to 0.0